test/mux-client-mode.sh

```diff
@@ -62,28 +46,16 @@ cleanup() {
 trap "cleanup" EXIT
 
 reset_fluentd_daemonset() {
-  muxcerts=`oc get daemonset logging-fluentd -o yaml | egrep muxcerts` || :
+  muxcerts=$( oc get daemonset logging-fluentd -o yaml | egrep muxcerts ) || :
 
   if [ "$muxcerts" = "" ]; then
-    oc get daemonset logging-fluentd -o yaml | sed '/volumes:/ a\
-      - name: muxcerts\
-        secret:\
-          defaultMode: 420\
-          secretName: logging-mux\
-' | oc replace -f -
-
-    oc get daemonset logging-fluentd -o yaml | sed '/volumeMounts:/ a\
-        - mountPath: /etc/fluent/muxkeys\
-          name: muxcerts\
-          readOnly: true\
-' | oc replace -f -
+      os::log::debug "$( oc set volumes daemonset/logging-fluentd --add --overwrite \
+               --name=muxcerts --default-mode=0400 -t secret -m /etc/fluent/muxkeys --secret-name logging-mux 2>&1 )"
   fi
 }
 
-os::log::info Starting mux-client-mode test at $( date )
```

Comment:
We could keep this "Starting mux-client-mode" line?

Reply:
It is redundant when running this test in CI, since entrypoint.sh also prints a similar message

Reply:
Ok.  Thanks.